main.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+#include "test/catch.hpp"
+#include "current_samples.h"
+
+rangeAndReadings rangeReadings;
+int sizeOfcurrentInAmpere;
+
+TEST_CASE("test current set 1") {
```

Comment:
Good set of test values 👍 
The test cases needs to be extended with single value and double value array ex {3} {3,3} and the outcome needs to be verified.

Reply:
done the changes